packages/mobile/src/send/ValidateRecipientIntro.tsx

```diff
@@ -61,45 +57,43 @@ class ValidateRecipientIntro extends React.Component<Props> {
     ValoraAnalytics.track(SendEvents.send_secure_start, { confirmByScan: false })
   }
 
-  render() {
-    const { t } = this.props
-    const { recipient } = this.props.route.params.transactionData
-    const displayName = getDisplayName(recipient, t)
-    const e164PhoneNumber = recipient.e164PhoneNumber
+  const AVATAR_SIZE = 64
```

Comment:
i think that this should stay as a constant outside the component, since the value doesn't change we don't need to recreate this on every rerender :)